### PR TITLE
Add ability for generate-config script to add directly to ESR

### PR DESCRIPTION
**Pull request by matteyu**
-Changed python interpreter from 2.7 to 3.6
-Added web3 functionality
-Changed logging to info
-Web3/Infura account keys read off external files

### CODE REVIEW

File: peerlist-server/generate-config.py [hunk 1]
```diff
@@ -1,17 +1,389 @@
...
 from argparse import ArgumentParser
 from argparse import RawDescriptionHelpFormatter
 
-logging.basicConfig(stream=sys.stdout, level=logging.DEBUG)
+logging.basicConfig(stream=sys.stdout, level=logging.INFO)
 logger = logging.getLogger(__name__)
 
+# required to have these files in the peerlist-server directory
+# infura = infura link including API key
+i = open("infura", "r")
```
Comment (amastracci) [inline peerlist-server/generate-config.py]: Do these need to be global?

File: peerlist-server/generate-config.py [hunk 2]
```diff
@@ -40,6 +412,37 @@ def get_node_uuid(node_id, working_directory):
 
 
 def make_peerlist_entry(uuid, node_id, same_port=False):
+    
+    ####Add node to BluzelleDockerSwarm
+    nonce = w3.eth.getTransactionCount(acct.address)
+    node_name = "node_{}".format(node_id)
+    node_host = ([l for l in ([ip for ip in socket.gethostbyname_ex(socket.gethostname())[2]
```
Comment (amastracci) [inline peerlist-server/generate-config.py]: Move this to function

Comment (amastracci) [reply peerlist-server/generate-config.py]: Give some thought to failure cases and where we see console errors within AWS

File: peerlist-server/generate-config.py (hunk 1 above)
Comment (matteyu) [reply peerlist-server/generate-config.py]: Yep.  They're required by the acct variable and contract_address variable which is used by multiple functions in the script.  It represents the ropsten address and ropsten contract address.

File: peerlist-server/generate-config.py (hunk 2 above)
Comment (matteyu) [reply peerlist-server/generate-config.py]: Moved.  Changes reflected with the new commit.

For viewing errors on console, it's a configuration through the cloudformation script to include CloudWatch.

**Pull request closed without merging**